pygmt/tests/test_grd2cpt.py

```diff
@@ -0,0 +1,344 @@
+"""
+Tests for grd2cpt.
+"""
+import os
+
+import numpy as np
+import pytest
+from pygmt import Figure
+from pygmt.datasets import load_earth_relief
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile
+from pygmt.helpers.testing import check_figures_equal
+from pygmt.src.grd2cpt import grd2cpt
+
+TEST_DATA_DIR = os.path.join(os.path.dirname(__file__), "data")
+POINTS_DATA = os.path.join(TEST_DATA_DIR, "points.txt")
+
+
+@pytest.fixture(scope="module", name="points")
```

Comment:
I think we can remove most tests, except:

- `test_grd2cpt`
- `test_grd2cpt_blank_output`
- `test_grd2cpt_invalid_output`
- `test_grd2cpt_output_to_cpt_file`
- `test_grd2cpt_unrecognized_data_type`
- `test_grd2cpt_categorical_and_cyclic`

Reply:
Removed in bf68c85